pagopa/pn-frontend
# Feat/pn 11970

## Short description
Removed context from legal contacts

## List of changes proposed in this pull request
- Created two new dialog
- Copied code from context to PecContactItem
- Removed context usage from PecContactItem
- Reworked tests

## How to test
- Login with pf/pg -> go to "Recapiti" -> add/modify/delete "default" legal contact and check the everything is ok
